frontend/src/APIClients/SimpleEntityAPIClient.ts

```diff
@@ -41,7 +41,7 @@ const create = async ({
     });
     return data;
   } catch (error) {
-    return error;
+    return error as SimpleEntityResponse;
```

Comment:
I am not a big fan of this but I am not sure what the better alternative is :/

Reply:
I spoke with the VP of Engineering, and they think the best thing here would be to throw the error instead of trying to return something. 

```
catch (error: unknown) {
    throw error;
 }

Reply:
Ah okay sounds good! This is definitely something that should get fixed in the starter code... it's weird to see a function returning its error instead of throwing it...

Reply:
Update: this makes eslint throw an error 🤨

Reply:
Ok I fixed the linter problems now :D
Linter doesn't love `catch (error) { throw error; }` - it says the try-catch block is unnecessary if we're just going to throw it without performing any other actions...